ci-operator/jobs/openshift/osde2e/openshift-osde2e-main-periodics.yaml

```diff
@@ -154,6 +154,61 @@ periodics:
     - name: osde2e-credentials
       secret:
         secretName: osde2e-credentials
+- agent: kubernetes
+  cluster: api.ci
+  cron: 0 */12 * * *
+  decorate: true
+  extra_refs:
+  - base_ref: main
+    org: openshift
+    repo: osde2e
+  labels:
+    pj-rehearse.openshift.io/can-be-rehearsed: "true"
+  name: osde2e-stage-aws-addon-ocs-converged
+  spec:
+    containers:
+    - args:
+      - test
+      - --secret-locations
+      - $(SECRET_LOCATIONS)
+      - --configs
+      - $(CONFIGS)
+      command:
+      - /osde2e
+      env:
+      - name: ADDON_IDS
+        value: ocs-operator
+      - name: ADDON_TEST_HARNESSES
+        value: quay.io/mkarnikredhat/ocsci-osd
+      - name: SECRET_LOCATIONS
+        value: /usr/local/osde2e-common,/usr/local/osde2e-credentials
+      - name: CONFIGS
+        value: aws,stage,addon-suite
+      - name: POLLING_TIMEOUT
+        value: "14400"
+      - name: NUM_WORKER_NODES
+        value: "9"
+      image: quay.io/app-sre/osde2e
+      imagePullPolicy: Always
+      name: ""
+      resources:
+        requests:
+          cpu: 10m
+      volumeMounts:
+      - mountPath: /usr/local/osde2e-common
+        name: osde2e-common
+        readOnly: true
+      - mountPath: /usr/local/osde2e-credentials
+        name: osde2e-credentials
+        readOnly: true
+    serviceAccountName: ci-operator
+    volumes:
+    - name: osde2e-common
+      secret:
+        secretName: osde2e-common
+    - name: osde2e-credentials
```

Comment:
This name and the secret name should match the defined secrets below (`ocs-converged-credentials`)